bin/setup_chef_perms.rb

```diff
@@ -0,0 +1,27 @@
+require 'rubygems'
+require 'chef/knife'
+require 'chef/server_api'
+
+knife_rb_path = File.join(ENV['HOME'], 'chef-bcpc', '.chef', 'knife.rb')
+config = File.open(knife_rb_path).read
+config.gsub!(/^client_key .*$/, "client_key '/etc/chef-server/admin.pem'")
+config.gsub!(/^node_name .*$/, "node_name 'admin'")
+
+#Chef::Config.from_file(knife_rb_path)
+Chef::Config.from_string(config, 'Modified knife.rb')
+
+options = {
+  client_name: 'admin',
+  raw_key: File.open('/etc/chef-server/admin.pem').read
+}
+
+rest = Chef::ServerAPI.new(Chef::Config[:chef_server_url])
+
+Chef::Node.list.each do |node|
```

Comment:
Since we setup the ACLs to be like the old chef-server, is this still needed?

Reply:
To allow the bootstrap host proper permissions I need this in [setup_chef_cookbooks.sh](https://github.com/bloomberg/chef-bach/pull/1206/files/73d22bfce559dacbb3ff7685655c228f06cd3b22#diff-95d29768b114a017d5a3307f0e57eb09R67)